src/controllers/slack.controller.ts

```
@@ -0,0 +1,165 @@
+
+    const updatedMessage = JSON.parse(JSON.stringify(originalMessage));
+    
+    if (updatedMessage.attachments && updatedMessage.attachments[0]) {
+      const newStatus = getNewStatusFromAction(action);
+      const statusColors = {
+        'resolved': 'good',
+        'ignored': 'warning',
+        'archived': '#808080',
+        'unresolved': 'danger',
+      };
+      
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)